src/lint/colors/tests/using-astro-color.ts

```diff
@@ -0,0 +1,82 @@
+import { FillStyleNode } from "../../../types/figma";
+import { LintingResult } from "../../../types/results";
+import { stripToLoadableId } from "../../../tokens";
+import { tokens } from "../../../tokens";
+const { colorTokens } = tokens();
+
+const usingAstroColor = (node: FillStyleNode): Promise<LintingResult> => {
+  const test = "Using an Astro Color";
+  const name = node.name;
+  const fillStyleId = node.fillStyleId;
+
+  return new Promise((resolve) => {
+    if (!fillStyleId) {
+      return resolve({
```

Comment:
Same suggestion as the above file to instantiate the object early on and then use its instance to spread into the parameter for resolve().